### ISSUE TITLE
Standard/CF variable names for litter

### ISSUE THREAD

**Issue by annethomas** [PecanProject/pecan#1487]
Litter output variables currently don't have a satisfactory standard naming system. They should match CF.

## Description
@mdietze wants the DALEC-associated variables LeafLitter and WoodLitter and another version, leaf_litter (in BETY) to be changed to reflect CF names litter_carbon_flux and litter_carbon_content. We also want to incorporate CF names subsurface_litter_carbon_content and surface_litter_carbon_content into the hierarchy. This is related to issue #1442 and hierarchical grouping of pool variables. 
@tonygardella you'll need to change the standard output table to include the cf variables, and we'll need to rename the variables in BETY where possible and fix the models that use the DALEC output names.

**Comment by annethomas** [comment]
@mdietze A few follow-up questions:
1. Since the DALEC names are split into Leaf and Wood, do we want to rename the variables using the CF convention, so leaf_litter_carbon_flux and wood_litter_carbon_flux, rather than changing something to the exact CF name? 
2. Then do we also want the aggregated variable litter_carbon_flux as a new variable?
3. Is litter_carbon_content replacing leaf_litter in BETY? Does that need to be split into leaf/wood at some point?
4. Are the surface and subsurface variables also new variables to be added?

**Comment by mdietze** [comment]
1. wouldn't a split of the litter flux be between leaf and fine root? That seems like a find distinction at the next level of the hierarchy. Woudn't the wood flux be CWD?
2. yes
3. yes, but that will take checking the code and the trait data before you change the names and units.  litter_carbon_content is already being split between surface and subsurface.
4. yes

**Comment by annethomas** [comment]
@mdietze That's right, I remember talking about coarse woody debris. It's named WoodyLitter (mis-typed that earlier) in the table but I'm not sure exactly what it means.

**Comment by mccabete** [comment]
@mdietze  Where in the code would I find where things are already being summed up? the model2netcdf?

**Comment by annethomas** [comment]
There's also a RootLitter variable, kg C m-2 s-2, in BETY

**Comment by dlebauer** [comment]
Is it possible to use the standard name fields instead of changing the
variable name / units in BETYdb? I think we added those fields for exactly
this reason.

**Comment by mdietze** [comment]
@dlebauer it's possible to use standard name in places, but it's also true that we sometimes come across names that could have been better thought out.  For example, we have one variable names ```leaf_litter``` and another named ```LeafLitter```. One is a flux, the other is a pool. One is g DM, the other is kg C. I looked at this and felt that these names and units really do need to change to be clear and consistent, and in this case there are CF names for these that are very clear (```litter_carbon_flux``` and ```litter_carbon_content```) and have consistent units. If this was just internally we could deal with standard name look-ups (though I do think it could make the code really ambiguous if we're using these as internal variable names), but we're also about to add these to the model input and output standards, which just feels like shooting ourselves in the foot.

**Comment by mccabete** [comment]
Here's a first pass at making this happen just using MSTMIP variables and the model outputs that are summed into MSTMIP variables in model2netcdf.  

[Hierarchical structure of variable names .pdf](https://github.com/PecanProject/pecan/files/1099019/Hierarchical.structure.of.variable.names.pdf)

Second step would be: Expanding this hierarchy to include all of the variables that can be reported from models (For FATES + CLM that is a little intimidating) 

Insuring that BETY variables are either included in this hierarchy or a removed for being redundant.

**Comment by annethomas** [comment]
for reference, Clark et al. 2001 NPP paper: NPP includes "aboveground biomass increment, fine litterfall, aboveground losses to consumers, emissions of biogenic volatile organic compounds (BVOCs), aboveground losses of leached organic compounds, net increments in biomass of coarse and fine roots, dead coarse and fine roots, root losses to consumers, root exudates, carbohydrates exported by plants to their mycorrhizal or nodule symbionts, and any net increases in stores of nonstructural carbohydrates"

**Comment by annethomas** [comment]
A diagram: 
http://onlinelibrary.wiley.com/doi/10.1890/1051-0761(2001)011%5B0356:MNPPIF%5D2.0.CO;2/full

**Comment by annethomas** [comment]
Here's FIA's field measurement protocol: https://www.fia.fs.fed.us/library/field-guides-methods-proc/docs/2016/core_ver7-1_10_2016-opt.pdf
